src/utils/bigNumber.ts

```diff
@@ -0,0 +1,11 @@
+import type { BigNumberish } from '@ethersproject/bignumber';
+import { commify, formatUnits, parseUnits } from '@ethersproject/units';
+
+export const parseToFormattedNumber = (
+  value: string | BigNumberish,
+  unit: BigNumberish = 'kwei'
+) => {
+  const bigNumber = typeof value === 'string' ? parseUnits(value, 0) : value;
```

Comment:
the units should be `3`.

```
export const DECIMAL_UNITS = 3;
export const FAUCET_AMOUNT = parseUnits('0.5', DECIMAL_UNITS);
export const MINT_AMOUNT = parseUnits('2000', DECIMAL_UNITS);
```
https://github.com/FuelLabs/swayswap/blob/51d4d1300242d784b72755b90513e4d4bc8fd69f/client/src/config.ts